plugins/dynatrace/src/components/Problems/ProblemsTable/ProblemsTable.tsx

```diff
@@ -63,19 +61,20 @@ export const ProblemsTable = ({ problems }: ProblemsTableProps) => {
       title: 'Start Time',
       field: 'startTime',
       render: (row: Partial<DynatraceProblem>) =>
-        new Date(row.startTime || 0).toString(),
+        new Date(row.startTime || 0).toLocaleString(),
```

Comment:
this pattern of `|| 0` is overall a bit unfortunate, should probably make a helper function instead that returns `N/A` or whatever if there's no timestamp :) This will just show 1970-01-01 won't it. But I presume this just doesn't happen, and just tries to appease typescript.

Reply:
you're right, I'll do that. It would just show epoch which isn't very clear for the user